src/test/java/tests/TestOpenPageChrome.java

```diff
@@ -32,4 +35,52 @@ public void openPage() {
 
         driver.quit();
     }
+
+    @Test
+    public void TestForShoppingCart() {
+        System.out.println("This is a test for debugging");
+        getPropertyManagerInstance();
+        WebDriver driver = WebDriverFactory.getWebDriver(BrowserName.CHROME);
+
+        String url = getProperty(BASE_URL);
+        driver.navigate().to(url);
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+
+        //TESTING BY DEBUG SHOPPING CART TABLE CLASS
```

Comment:
Just some thoughts for the future: 
why do we create so many methods and don't use them already in some real, valid tests? 
There's a chance we are creating methods that won't be used at all or might be faulty. 
It is also pretty hard to review as well - because it's hard to imagine how actual tests would look like.